codereadyworkspaces/stack/Dockerfile

```diff
@@ -1,34 +1,77 @@
-FROM quay.io/crw/plugin-java8-rhel8:2.11
-
-ENV ARGOCD_VERSION=2.0.5
-ENV YQ_VERSION=4.11.2
-ENV HELM_VERSION=3.6.3
-ENV KUBESEAL_VERSION=0.13.1
-ENV TEKTON_VERSION=0.19.1
-ENV OC_VERSION=4.8.3
-ENV JQ_VERSION=1.6
-ENV CONFTEST_VERSION=0.23.0
-ENV ZSH_VERSION=5.8
+FROM registry.redhat.io/ubi8/ubi:8.4
 
 USER root
 
+# dnf repos
+RUN rm -f /etc/yum.repos.d/*
+ADD ubi8.repo /etc/yum.repos.d/ubi8.repo
+RUN dnf install -y centos-gpg-keys --nogpgcheck && \
+    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial
+
+# rht-labs stuff
+ENV ARGOCD_VERSION=2.0.5 \
+    YQ_VERSION=4.11.2 \
+    HELM_VERSION=3.6.3 \
+    KUBESEAL_VERSION=v0.13.1 \
```

Comment:
can you make this `KUBESEAL_VERSION=0.13.1` to comply with the others? it'd also fail in check action with v